server/lib/meetup.js

```diff
@@ -16,18 +16,33 @@ class Meetup {
     };
   }
 
-  makeHeadersForTier(tiername) {
-    let header = '', usersList = '';
+  /**
+   * We show the 5 top sponsors (if any)
+   * Otherwise we show the 10 top backers
+   */
+  makeHeader() {
+    let header = '', usersList = '', tier = {};
+    const users = values(this.group.users);
 
-    const usersInTier = filter(values(this.group.users), { tier: tiername });
+    // Order backers by totalDonations DESC
+    users.sort((a, b) => b.totalDonations - a.totalDonations);
 
-    if (usersInTier.length > 0) {
-      usersList = usersInTier.map((user) => (user.website) ? `<a href="${user.website}">${user.name}</a>` : user.name).join(', ');
+    const sponsors = filter(users, { tier: 'sponsor' }).slice(0, 5); // max 5 sponsors
+    const backers = users.slice(0, 10); // or max 10 backers
+
+    if (sponsors.length > 0) {
+      tier = { name: 'sponsor', users: sponsors };
+    } else {
+      tier = { name: 'backer', users: backers };
+    }
+
+    if (tier.users.length > 0) {
+      usersList = tier.users.map((user) => (user.website) ? `<a href="${user.website}">${user.name}</a>` : user.name).join(', ');
       usersList = usersList.replace(/,([^,]*)$/,' and$1');
-      header += `<p>Thank you to our sponsors ${usersList}</p>\n`;
+      header += `<p>Thank you to our ${tier.name}s ${usersList}</p> `;
```

Comment:
"thank you to" doesn't seem valid English grammar to me. ("thank you" is shorthand for "I thank you")